### PR TITLE
[docs] Improve Website Styling

### PR DESCRIPTION
**Pull request by anmol372**
**Description of the change:**
Attempted to increase readability by
1. Increase the size of headings,
2. Added styling to the left-nav bar
3. Corrected doc names and subsequent links for the correct generation of HTML pages.
4. Added Breadcrumb Navigation Styling.

**Motivation for the change:**
- Improve look and feel.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by anmol372** [comment]
/hold

Should be merged after `[4002](https://github.com/operator-framework/operator-sdk/pull/4002)`

**Comment by camilamacedo86** [comment]
Could you please rebase it with the master? Otherwise, we cannot move forward with it.

**Comment by anmol372** [comment]
@camilamacedo86 done rebasing.

**Comment by asmacdo** [comment]
The left bar no longer collapses. That's a dealbreaker for me.

Screenshot from 2020-11-02 13-24-09

**Comment by anmol372** [comment]
> The left bar no longer collapses. That's a dealbreaker for me.

I agree, right now it collapses on clicking on any subtopics. I am working on getting it to collapse from the get-go.

File: website/content/en/docs/faqs/_index.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ You should not have separate logic. Instead design your reconciler to be idempot
 
 Use a [finalizer].
 
-## I keep seeing the following intermittent warning in my Operator's logs: `The resourceVersion for the provided watch is too old.` What's wrong?
+## I keep seeing the following intermittent warning in my Operator's logs: 
+`The resourceVersion for the provided watch is too old.` 
+## What's wrong?
```
Comment (estroz) [inline website/content/en/docs/faqs/_index.md:21]: Is it possible to use the old header, maybe something more succinct like
```suggestion
## I see the warning in my Operator's logs "the resourceVersion for the provided watch is too old". What's wrong?
```

File: website/content/en/docs/building-operators/golang/quickstart.md [hunk 2]
```diff
@@ -87,6 +87,7 @@ Read the [tutorial][tutorial] for an in-depth walkthough of building a Go operat
 [go_tool]:https://golang.org/dl/
 [docker_tool]:https://docs.docker.com/install/
 [kubectl_tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
-[operator_install]: /docs/installation/install-operator-sdk
+[operator_install]: /docs/installation/
+[envtest-setup]: /docs/building-operators/golang/references/envtest-setup
```
Comment (estroz) [inline website/content/en/docs/building-operators/golang/quickstart.md:91]: I don't see this link being used anywhere
```suggestion
```

File: website/assets/scss/_breadcrumb.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.breadcrumb {
+    display: flex;
+    flex-wrap: wrap;
+    //padding: $breadcrumb-padding-y;
```
Comment (estroz) [inline website/assets/scss/_breadcrumb.scss:4]: Can these commented CSS directives be removed

File: website/content/en/docs/building-operators/golang/quickstart.md (hunk 2 above)
Comment (anmol372) [reply website/content/en/docs/building-operators/golang/quickstart.md:91]: The element this links to got removed in [3983](https://github.com/operator-framework/operator-sdk/pull/3983), I'll clean up the trailing link in my PR as well.

File: website/content/en/docs/faqs/_index.md (hunk 1 above)
Comment (anmol372) [reply website/content/en/docs/faqs/_index.md]: Screen Shot 2020-11-10 at 2 38 42 PM

Updated formatting style to the previous header.

File: website/assets/scss/_breadcrumb.scss (hunk 3 above)
Comment (anmol372) [reply website/assets/scss/_breadcrumb.scss]: Done

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**